docs/modules/general/proto.mdx

```diff
@@ -0,0 +1,335 @@
+          <br />
+          Everything scores zero points by default.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <label>1.3.5</label>
+        </td>
+        <td>Filters is aware of who owns TNT.</td>
+      </tr>
+      <tr>
+        <td>
+          <label>1.3.4</label>
+        </td>
+        <td>Wools have a location property.</td>
+      </tr>
+      <tr>
+        <td>
+          <label>1.3.3</label>
+        </td>
+        <td>Define how overlapping regions should behave.</td>
+      </tr>
+      <tr>
+        <td>
+          <label>1.3.2</label>
+        </td>
+        <td>Added monument modes.</td>
+      </tr>
+      <tr>
+        <td>
+          <label>1.3.1</label>
+        </td>
+        <td>Off-by-one region bug fixed.</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+## Map Protocol Changelog
+
+### Changes in 1.4.2
+
+<span className="badge badge--danger">Breaking</span>
+<br />
+
+- The objectives filter will now always automatically derive team ownership for the objective from context. To match an objective without any specific team, the <label>any="true"</label> attribute needs to be specified.
+  ***
+
+<span class="badge badge--success">New</span>
+<br />
+
+<span className="badge badge--secondary">Partially implemented</span>{" "}
+
+- Portals, kits, score boxes & structures can now be dynamically applied using dynamic filters. All region types are dynamic, however, only some filter types are dynamic.
+  - Kits now have [`give`, `take` & `lend` properties](/docs/modules/gear/kits#dynamic-kits).
+  - Portals have [`forward`, `reverse` & `transit` properties](/docs/modules/mechanics/portals#portal-sub-elements).
+  - Structures & [score boxes](/docs/modules/objectives/scoring#score-box-attributes) have a `trigger` property.
+- <span className="badge badge--secondary">Not implemented</span> Added a player
+  count filter.
+- Added a `<grounded/>` filter to check if the player is standing on the ground.
+- Added `<match-started/>`, `<match-running/>` & `<match-finished/>` filters that are used to check the state of the current match.
+- New `pre-match-physics` attribute for the terrain module to enable physics events before the match starts.
+
+### Changes in 1.4.1
+
+<span className="badge badge--danger">Breaking</span>
+<br />
+
+<span className="badge badge--secondary">Not implemented</span>
+
+- `<random>` filters now only respond to instantaneous events, and abstain when used in any persistent context. This breaks some uses that relied on undefined behavior, but these would likely break anyway as filters gain more dynamic functionality.
+  ***
+
+<span class="badge badge--primary">Change</span>
+<br />
+
+- <span className="badge badge--secondary">Not implemented</span> As of Minecraft
+  1.9, TNT fuse time is no longer limited to 4 seconds.
+- Portals react to player move events allowing almost instant teleportation as soon as the filter matches.
```

Comment:
This only applies to dynamic portals. Standard portals still only check if a player is trying to enter its region.